### ISSUE TITLE
Updated release for newer versions of the shim

### ISSUE THREAD

**Issue by jpflueger** [KWasm/kwasm-node-installer#53]
Hello all! Would it be possible to tag a new release with the updated versions of the wasm shims? It looks like the last tagged "release" is for v0.3.0 (i.e. v0.9.1 shims) but main has support for v0.10.0 shims. Happy to also move this request to the operator repo if that's preferable. Thanks!

**Comment by 0xE282B0** [comment]
Sure, we can do a release with updated pinned versions tomorrow.

**Comment by 0xE282B0** [comment]
Hi @jpflueger,
The helm chart of the operator uses the latest installer image from the [`main` branch](https://github.com/KWasm/kwasm-operator/blob/main/charts/kwasm-operator/values.yaml#L19).
I just released [the node-installer v0.3.1](https://github.com/KWasm/kwasm-node-installer/releases/tag/v0.3.1). To use exactly this version with the operator helm chart, you can override the installer image:
```bash
helm upgrade --install -n kwasm --create-namespace kwasm-operator kwasm/kwasm-operator \
  --set kwasmOperator.installerImage="ghcr.io/kwasm/kwasm-node-installer:v0.3.1"
```
